manifests/install/repos/extra.pp

```diff
@@ -30,12 +30,7 @@
   }
 
   if $ipa_authentication and $configure_ipa_repo {
-    yumrepo { 'adelton-identity':
-      enabled  => 1,
-      gpgcheck => 0,
-      baseurl  => "http://copr-be.cloud.fedoraproject.org/results/adelton/identity_demo/epel-${osreleasemajor}-\$basearch/",
-      before   => Package['mod_authnz_pam', 'mod_lookup_identity', 'mod_intercept_form_submit', 'sssd-dbus'],
-    }
+    notify {"The --foreman-configure-ipa-repo option is deprecated and no longer needed. If you have used it before, disable and delete the repo.":}
```

Comment:
you'll have to use single quotes here (that's the reason why automated check fails), double qoute string is used when there's some $variable that should be interpolated